example/lib/pages/login_page.dart

```diff
@@ -58,27 +58,39 @@ class LoginPage extends HookWidget {
                   }
                 },
               ),
-              
               ElevatedButton(
                 child: const Text('Create Random Wallet'),
                 onPressed: () async {
                   try {
                     print('Creating a random wallet...');
-                    var signer = EthPrivateKey.createRandom(Random.secure()).asSigner();
-                    print('Wallet created with address: ${await signer.address}');
+                    var signer =
+                        EthPrivateKey.createRandom(Random.secure()).asSigner();
+                    print(
+                        'Wallet created with address: ${await signer.address}');
                     print('Authorizing the wallet...');
                     bool isAuthorized = await session.authorize(signer);
                     print('Wallet authorized: $isAuthorized');
-                    print(context);
-                    print('Session: $session');
-                    print('Context: $context');
+                    if (isAuthorized) {
+                      await session.loadSaved();
+                    }
```

Comment:
The check in the router always fails since we never rerun the loadSaved() function.

Router check that was failing and redirecting to `/login` 
 https://github.com/lecksfrawen/xmtp-flutter/blob/5e47639a555780425459e839ba56b52a1c2923d5/example/lib/router.dart#L13 
 
 Code that wasn't running before:
 https://github.com/lecksfrawen/xmtp-flutter/blob/5e47639a555780425459e839ba56b52a1c2923d5/example/lib/session/foreground_session.dart#L102-L118